src/makeRobotAccountant.js

```diff
@@ -7,6 +7,19 @@
 
 function makeRobotAccountant() {
   // write code here
+  let counter = 0;
+
+  return function getSum(a) {
+    return function inner(b) {
+      counter++;
+
+      if (counter % 2 === 0 && counter > 3) {
```

Comment:
The calculation for determining whether the call is even is incorrect. The count should be checked for divisibility by 4, not 2, to alternate between correct sum and error after the third call.